generators/dart/dart_generator.js

```diff
@@ -0,0 +1,311 @@
+/**
+ * @license
+ * Copyright 2014 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview Helper functions for generating Dart for blocks.
+ * @suppress {checkTypes|globalThis}
+ */
+
+import * as goog from '../../closure/goog/goog.js';
+goog.declareModuleId('Blockly.Dart');
+
+import * as Variables from '../../core/variables.js';
+import * as stringUtils from '../../core/utils/string.js';
+// import type {Block} from '../../core/block.js';
```

Comment:
Can we clean up these commented out exports?

Reply:
The lines (in uncommented form) were generated from `goog.requireType` calls by the `js2ts` migration script.  Because `import type` is not a thing in plain JS, they can't be left in, but they will be needed when these files are migrated to TypeScript, so I thought it most sensible to just comment them out for now.